ddclient

```diff
@@ -1475,8 +1475,15 @@ sub test_possible_ip {
 
     {
         local $opt{'use'} = 'if';
-        foreach my $if (grep {/^[a-zA-Z]/} `ifconfig -a`) {
-            $if =~ s/:?\s.*//is;
+        # Note: The `ip` command adds a `@eth0` suffix to the names of VLAN
+        # interfaces. That `@eth0` suffix is NOT part of the interface name.
+        my @ifs = map({ /^[^\s:]*:\s*([^\s:@]+)/ ? $1 : () }
+                      `command -v ip >/dev/null && ip -o link show`);
```

Comment:
```
command -v ip >/dev/null && ip -o link show
Dump was interrupted and may be inconsistent.
....
a gazillion networks
....
```

I got this during testing. Not sure what it means and I couldn't find something on the net. Do we need to worry about this?

Reply:
I would like to understand what happened in case it's a new bug. Were you using `-debug` and/or `-verbose`?  Note that `-debug` causes it to continue to attempt the update.

Reply:
I just put the command into my shell and I think I know what happend: I just have to many rules that change sometimes.